doc/developer/guide.md

```diff
@@ -85,17 +85,7 @@ Confluent Platform, as it is a rather heavy dependency. Most Materialize
 employees, or other major contributors, will probably need to run the full test
 suite and should therefore install the Confluent Platform.
 
-#### All platforms
-
-First, install the CLI. As of early July 2022 you can run this command on
-macOS and Linux:
-
-```shell
-curl -sL --http1.1 https://cnfl.io/cli | sudo sh -s -- -b /usr/local/bin latest
-```
-
-If this no longer works, follow the instructions in the [Confluent CLI]
-documentation. Then please update this guide with the new instructions!
+The latest versions of the Confluent CLI do not include all 
```

Comment:
I think you picked up some stray WIP changes about dealing with the latest version of the local Confluent CLI!

Reply:
🤦